app/components/radio-group/component.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import layout from './template';
+
+
+export default Ember.Component.extend({
+  layout,
+  options: null,
+  value: null,
+  labelLeft: null,
+  labelRight: null,
```

Comment:
Good call! I also replaced `labelLeft` and `labelRight` with just `labels`, so I will make those changes.